benefits/enrollment/views.py

```diff
@@ -45,8 +46,20 @@ def token(request):
             audience=payment_processor.audience,
         )
         client.oauth.ensure_active_token(client.token)
-        response = client.request_card_tokenization_access()
-        session.update(request, enrollment_token=response.get("access_token"), enrollment_token_exp=response.get("expires_at"))
+
+        try:
+            response = client.request_card_tokenization_access()
+        except Exception as e:
+            if isinstance(e, HTTPError) and e.response.status_code >= 500:
+                sentry_sdk.capture_exception(e)
+                data = {"redirect": reverse(ROUTE_SYSTEM_ERROR)}
+                return JsonResponse(data)
+            else:
+                raise e
```

Comment:
Send the same analytic event class as above. It should include the error code.

Reply:
Will be done for 400-level error case of `/token` (see #2032 acceptance criteria)